src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -748,4 +749,11 @@ public static void onCommandRegister(CommandDispatcher<CommandSource> dispatcher
         MinecraftForge.EVENT_BUS.post(evt);
         return evt;
     }
+    
+    public static EntityEmittedSoundEvent entityEmittedSound(Entity source, double x, double y, double z, SoundEvent soundIn, SoundCategory category, float volume, float pitch)
+    {
+        EntityEmittedSoundEvent evt = new EntityEmittedSoundEvent(source, new net.minecraft.util.math.vector.Vector3d(x, y, z), soundIn, category, volume, pitch);
```

Comment:
As this is a Forge class, this use of a fully qualified name should be replaced with an import.